website/docs/configuration/recommended-rbac-configuration.mdx

```diff
@@ -178,3 +180,142 @@ roleRef:
+
+Weave GitOps Enterprise integrates with Flagger in order to provide a view on progressive delivery deployments. Part of that view, is the ability to view all the resources that Flagger manages during its operation. The default ClusterRole `gitops-canaries-reader`, used for viewing canary details, includes the minimum permissions necessary for a user to be able to view canary object details, metric template object details and canary related events. When a canary relies on a service mesh such as Linkerd or Istio for the rollout, then this ClusterRole needs to be extended so that it can read the additional service mesh resources being generated by Flagger. Note that this functionality relies on each canary resource explicitly specifying the provider that it uses. The following table provides a list of all the custom resources that Flagger generates grouped by provider:
+
+| Provider          | API Group                 | Resource        |
+| ---               | ---                       | ---             |
+| AppMesh           | appmesh.k8s.aws           | virtualnode     |
+|                   | appmesh.k8s.aws           | virtualrouter   |
+|                   | appmesh.k8s.aws           | virtualservice  |
+| Linkerd           | split.smi-spec.io         | trafficsplit    |
+| Istio             | networking.istio.io       | destinationrule |
+|                   | networking.istio.io       | virtualservice  |
+| Contour           | projectcontour.io         | httpproxy       |
+| Gloo              | gateway.solo.io           | routetable      |
+|                   | gloo.solo.io              | upstream        |
+| Nginx             | networking.k8s.io         | ingress         |
+| Skipper           | networking.k8s.io         | ingress         |
+| Traefik           | traefik.containo.us       | traefikservice  |
+| Open Service Mesh | split.smi-spec.io         | trafficsplit    |
+| Kuma              | kuma.io                   | trafficroute    |
+| GatewayAPI        | gateway.networking.k8s.io | httproute       |
+
+For example, the following manifest shows how `gitops-canaries-reader` has been extended to allow the user for viewing TrafficSplit resources when Linkerd is used:
+
+```yaml title="gitops-canaries-reader.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gitops-canaries-reader
+rules:
+- apiGroups:
+  - flagger.app
+  resources:
+  - canaries
+  - metrictemplates
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - watch
+  - list
+# Additional permissions for Linkerd resources are added below
+- apiGroups:
+  - split.smi-spec.io
+  resources:
+  - trafficsplits
+  verbs:
+  - get
+  - list
+```
+
+### Setting up remote cluster permissions
+
+In order to be able to view remote cluster information, such as canaries in a remote cluster, from the management cluster, you need to take into account the following:
```

Comment:
> In order to be able to view remote cluster information, such as canaries in a remote cluster, from the management cluster,

Simplify - In order to view canaries in a remote cluster from the management cluster

Reply:
Updated 👍